nautobot/virtualization/migrations/0003_auto_slug.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.13 on 2021-08-05 03:23
```

Comment:
This will need to be renamed to 0004 now so that it doesn't conflict with the 0003 migration added in #821. Sorry for the inconvenience!